test/infrastructure/docker/api/v1beta1/devcluster_types.go

```diff
@@ -28,6 +28,46 @@ const (
 	ListenerAnnotationName = "inmemorycluster.infrastructure.cluster.x-k8s.io/listener"
 )
 
+// DevCluster's v1Beta2 conditions that apply to all the supported backends.
+
+// DevCluster's Ready condition and corresponding reasons that will be used in v1Beta2 API version.
+const (
+	// DevClusterReadyV1Beta2Condition is true if the DevCluster's VMProvisioned, NodeProvisioned,
+	// EtcdProvisioned (if present) and APIServerProvisioned (if present) conditions are true.
+	DevClusterReadyV1Beta2Condition = clusterv1.ReadyV1Beta2Condition
+
+	// DevClusterReadyV1Beta2Reason surfaces when the DevCluster readiness criteria is met.
+	DevClusterReadyV1Beta2Reason = clusterv1.ReadyV1Beta2Reason
+
+	// DevClusterNotReadyV1Beta2Reason surfaces when the DevCluster readiness criteria is not met.
+	DevClusterNotReadyV1Beta2Reason = clusterv1.NotReadyV1Beta2Reason
+
+	// DevClusterReadyUnknownV1Beta2Reason surfaces when at least one DevCluster readiness criteria is unknown
+	// and no DevCluster readiness criteria is not met.
+	DevClusterReadyUnknownV1Beta2Reason = clusterv1.ReadyUnknownV1Beta2Reason
```

Comment:
Is it intentional that none of these are used in this PR? Will we use them in a subsequent one?

Reply:
I think the idea was to use these 3 in dockercluster_backend.go l.177+

Reply:
Fixed